src/components/file-editor/FileEditor.tsx

```diff
@@ -19,7 +19,9 @@ const FileEditor: FC<FileEditorProps> = ({ onEdit, value }) => {
         toolbar:
           'undo redo | blocks fontsize | bold italic underline strikethrough | ltr rtl | link image media table mergetags | align lineheight | tinycomments | checklist numlist bullist indent outdent accordion | removeformat',
         content_style:
-          'body { font-family:Helvetica,Arial,sans-serif; font-size:14px }'
+          '@import url("https://fonts.googleapis.com/css2?family=Rubik:wght@400;600;800&display=swap"); body { font-family:Rubik; font-size:14px }',
+        font_family_formats:
+          'Andale Mono=andale mono,times; Arial=arial,helvetica,sans-serif; Arial Black=arial black,avant garde; Book Antiqua=book antiqua,palatino; Comic Sans MS=comic sans ms,sans-serif; Courier New=courier new,courier; Georgia=georgia,palatino; Helvetica=helvetica; Impact=impact,chicago; Symbol=symbol; Tahoma=tahoma,arial,helvetica,sans-serif; Terminal=terminal,monaco; Times New Roman=times new roman,times; Trebuchet MS=trebuchet ms,geneva; Verdana=verdana,geneva; Webdings=webdings; Wingdings=wingdings,zapf dingbats; Rubik=rubik'
```

Comment:
Seems like there is a list of key-value pairs with font name in editor and it`s actual value, separated by semicolons. Would it be better if we move key value pairs into separate object and use joining on it like following:

```jsx
const availableEditorFonts = {
     "Andale Mono": "andale mono"
}

font_family_formats: Object.entries(availableEditorFonts).map(entry => entry.join("=")).join("; ")
```

What do you think about this approach? Maybe adding or removing fonts will be more convenient using object

Reply:
Thanks for the suggestion 👍  I decided to move all the configurations to the constants file. Hopefully, it will be easier to maintain.